script.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener("DOMcontentLoaded", function(){
+createBoard(32);
+getSize();
+console.log("hi")
```

Comment:
Why do you still have console.log here?

Reply:
> Why do you still have console.log here?

Ooops! 
it will be removed immediately.